examples/apps/branding/app.py

```diff
@@ -0,0 +1,221 @@
+        margin=10,
+    ),
+    pmui.Paper(
+        pn.pane.Image(
+            "https://cdn.britannica.com/94/192794-050-3F3F3DDD/panels-electricity-order-sunlight.jpg",
+            margin=0,
+            sizing_mode="stretch_both",
+        ),
+        "# Reports\n\nDownload our annual report and see all our reports and presentations",
+        width=300,
+        sizing_mode="fixed",
+        margin=10,
+    ),
+    sizing_mode="fixed",
+    align="center",
+    margin=25,
+)
+
+
+@pn.cache
+def get_data(n_categories: int, n_rows=100):
+    x = np.random.rand(n_rows)
+    y = np.random.rand(n_rows)
+
+    categories = [f"Category {i+1}" for i in range(n_categories)]
+    category = np.random.choice(categories, n_rows)
+
+    dataframe = pd.DataFrame({"x": x, "y": y, "category": category}).sort_values(
+        "category"
+    )
+
+    return dataframe
+
+
+def get_categorical_plot(dark_theme: bool=False):
+    df = get_data(n_categories=3, n_rows=30)
+    cmap = colors.get_categorical_palette(n_colors=3, dark_theme=dark_theme)
+    return df.hvplot.scatter(
+        x="x",
+        y="y",
+        size=75,
+        color="category",
+        cmap=cmap,
+        height=350,
+        responsive=True,
+        title="Categorical Plot",
+        tools=["fullscreen"],
+        legend="top_right",
+    ).opts(backend_opts={"plot.toolbar.autohide": True}, toolbar="above")
+
+
+def get_continous_plot(dark_theme):
+    np.random.seed(42)
+    data = pd.DataFrame(
+        {
+            "x": np.random.rand(100),
+            "y": np.random.rand(100),
+            "value": np.random.rand(100) * 100,
+        }
+    )
+    cmap = colors.get_continous_cmap(dark_theme)
+    return data.hvplot.scatter(
+        x="x",
+        y="y",
+        c="value",
+        size=75,
+        cmap=cmap,
+        colorbar=True,
+        height=350,
+        responsive=True,
+        title="Continuous Plot",
+        tools=["fullscreen"],
+    ).opts(backend_opts={"plot.toolbar.autohide": True}, toolbar="above")
+
+categorical_plot = pn.bind(get_categorical_plot, dark_theme=state.param.dark_theme)
+continous_plot = pn.bind(get_continous_plot, dark_theme=state.param.dark_theme)
+
+page = pmui.Page(
+    header=[pn.HSpacer(), open_drawer],
+    sidebar=[
+        pn.pane.Image(
+            assets.VISION_PATH,
+            margin=(20, 10, 10, 10),
+            sizing_mode="scale_width",
+        ),
+        pmui.Column(
+            "# Settings",
+            state,
+            pn.VSpacer(min_height=25),
+            "### References",
+            github_link,
+            docs_link,
+            sizing_mode="stretch_both",
+            margin=(10, 10),
+            styles={"border": "1px solid gray"},
+        ),
+    ],
+    sidebar_width=425,
+    main=[pmui.Container("<h2>Buttons</h2>", example_buttons, "<h2>Plots</h2>", categorical_plot, continous_plot, drawer)],
+)
+
+page.param.dark_theme.rx.watch(lambda value: state.param.update(dark_theme=value))
```

Comment:
Is there a way to "get hold" of dynamic `dark_theme` value and bind to it without having yet instantiated the `page`?